dockerhub.py

```diff
@@ -278,14 +284,15 @@ def _do_request(self, method, address, **kwargs):
             raise ConnectionError('Connection Error. Download failed: {0}'.format(e))
         else:
             if resp.status_code == 401 and ttl > 0:
-                scope = None
                 # Update the auth token with the scope, and try again
                 try:
-                    scope = resp.headers['Www-Authenticate'].split(',')[2]
-                    scope = scope.split('=')[1].replace("\"", "")
+                    # Parse the Www-Authenticate line, looks like:
+                    # Bearer realm="https://git.ligo.org/jwt/auth",service="container_registry",scope="repository:lscsoft/lalsuite/lalsuite-v6.53:pull",error="invalid_token"
+                    reg=re.compile('(\w+)[=] ?"?([\w\:\/\.\-]+)"?')
+                    values = dict(reg.findall(resp.headers['Www-Authenticate']))
                 except:
```

Comment:
`except: pass`? really?
All that will cause is for 295 to blow up with a NameError on `values`.

Reply:
Probably used for debugging.  Removing.